criu/pie/restorer.c

```diff
@@ -1243,6 +1244,73 @@ static inline int restore_child_subreaper(int child_subreaper)
 		return 0;
 }
 
+static int fd_poll(int inotify_fd)
+{
+	struct pollfd pfd = {inotify_fd, POLLIN, 0};
+#ifdef SYS_poll
+	return sys_poll(&pfd, 1, 0);
+#else
+	struct timespec tmo = {0, 0};
+
+	return sys_ppoll(&pfd, 1, &tmo, NULL, sizeof(sigset_t));
```

Comment:
Why we can't use ppoll in all cases?

Reply:
Replaced sys_poll with sys_ppoll completely. Thanks, didn't even thought about it, now it looks better.